src/test/java/org/example/integration/AuthControllerIntegrationTest.java

```diff
@@ -0,0 +1,138 @@
+package org.example.integration;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import io.dropwizard.configuration.ResourceConfigurationSourceProvider;
+import io.dropwizard.testing.junit5.DropwizardAppExtension;
+import io.dropwizard.testing.junit5.DropwizardExtensionsSupport;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import javax.ws.rs.client.Entity;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import org.example.TrueApplication;
+import org.example.TrueConfiguration;
+import org.example.cli.Login;
+import org.example.client.DatabaseConnectionException;
+import org.example.db.DatabaseConnector;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+/**
+ * The type Auth controller integration test.
+ */
+@ExtendWith(DropwizardExtensionsSupport.class)
+public class AuthControllerIntegrationTest {
+  private final String baseUrl = System.getenv("BASE_URL");
+  private final String apiLogin = baseUrl + "/login";
+  private static final String email = System.getenv("ADMIN_EMAIL");
+  private static final String password = System.getenv("ADMIN_PASSWORD");
+
+  /**
+   * The App.
+   */
+  static final DropwizardAppExtension<TrueConfiguration> APP = new DropwizardAppExtension<>(
+      TrueApplication.class, null,
+      new ResourceConfigurationSourceProvider()
+  );
+
+  private static void addAccount() throws SQLException, DatabaseConnectionException {
```

Comment:
Would this be better named as `addAdminAccount` or to take the role as a parameter?